compiler-core/src/error.rs

```diff
@@ -1083,6 +1089,17 @@ Please make sure internal types do not appear in public functions and try again.
                 location: None,
             }],
 
+            Error::CannotPublishLeakedDevDependency { module, package } => {
+                let text = &format!("The module `{module}` is being imported from `{package}`, but package is not a direct dependency of your project.\nRun this command to add it to your dependencies:\n\tgleam add {package}");
+                vec![Diagnostic {
+                    title: "Developer dependency imported".into(),
```

Comment:
`Development dependency` is "more" gender neutral (in non English contexts)

Reply:
Fixed!